torch/_inductor/scheduler.py

```diff
@@ -2189,6 +2195,11 @@ def codegen(self):
 
             self.available_buffer_names.update(node.get_names())
 
+            if device is not None:
```

Comment:
We discussed in the meeting that flushing delineates kernels for the cpp backend, the approach we agreed on was to change this to an API `ready_to_flush` to signal to the scheduler that it should flush early.

Reply:
@mlazos the discussion is implemented in code.